massdriver/provider.go

```diff
@@ -32,8 +32,9 @@ func Provider() *schema.Provider {
 			},
 		},
 		ResourcesMap: map[string]*schema.Resource{
-			"massdriver_artifact":      resourceArtifact(),
-			"massdriver_package_alarm": resourcePackageAlarm(),
+			"massdriver_artifact":       resourceArtifact(),
+			"massdriver_available_cidr": resourceAvailableCidr(),
```

Comment:
Combining the providers becomes difficult with the `cache` provider since it requires the low-level `terraform-plugin-go` framework, whereas `jq` provider and this `available_cidr` resource use the much simpler `terraform-plugin-sdk` framework. This is because the the cache resource needs access to the [`DynamicPseudoType`](https://github.com/massdriver-cloud/terraform-provider-cache/blob/main/cache/provider.go#L60) which is only available in the lower level framework, which allows the resource to accept and cache any variable type.

I'll  move this out to its own `utility` provider.

Reply:
Yeah we can leave the cache out then but if we had a utility provider that we can slap more complicated functions into that would be awesome and really useful for non-md folk